test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/CheckAllAnswersAndSubmitControllerSpec.scala

```diff
@@ -161,7 +177,8 @@ class CheckAllAnswersAndSubmitControllerSpec
 
         "display the page" when {
 
-          def test(sessionData: SessionData, expectedTitleKey: String, userType: Option[UserType]): Unit = {
+          def test(sessionData: SessionData, expectedTitleKey: String, userType: Option[UserType], isATrust: Boolean)
```

Comment:
do you need the `isATrust` flag here - shouldn't what we had before work with `ompleteFillingOutReturn.subscribedDetails.isATrust`? That is essentially what you're passing in anyway

Reply:
It works for first test as it's just using existing `completeFillingOutReturn`. But does not for next test. as we are modifying `completeFillingOutReturn` with  `userType'

Reply:
it took me time to figure out this issue